src/utils.js

```diff
@@ -22,6 +24,17 @@ const isInvalidValidPassword = password => {
   }
 };
 
+/**
+ * @desc get hash from message then sign the hash
```

Comment:
why "message"? We the dapp data and not a message

Reply:
done